MVCDemos/Models/SampleData/ContextMenuData.cs

```diff
@@ -1,1243 +1,360 @@
 ﻿using System;
 using System.Collections.Generic;
 using System.Linq;
+using static System.Net.Mime.MediaTypeNames;
```

Comment:
Why it requires here?

Reply:
[4a3cf45](https://github.com/DevExpress/devextreme-demos/pull/3211/commits/4a3cf450c084b9e548b9cab85debcc7d1b60e319)
tested this one we can safely remove the unecessary imports even the one on top of it the Linq